src/bsp/src/posix/src/BSP.cpp

```diff
@@ -50,7 +54,11 @@ void BSP::initChip(void* args) {
 
     m_rosNodeHandle = std::shared_ptr<ros::NodeHandle>(new ros::NodeHandle("~"));
 
-    xTaskCreate(rosWatcher, "ros_watch", configMINIMAL_STACK_SIZE, NULL, tskIDLE_PRIORITY + 1,
+    TCPUartMock& tcpUart = static_cast<TCPUartMock&>(BSPContainer::getHostUart());
+    int port = m_rosNodeHandle->param("uart_mock_port", 0);
```

Comment:
Is it better to define "uart_mock_port" somewhere else than hard coded? The comment goes for "hive_mind" at line 53 as well

Reply:
@xgroleau We should have a string factory somewhere. Do we put that in the main cleanup task?

Reply:
Or more like getters in the ros BPS implementation? But yea that could go in the main cleanup